episodes/03-skimage-images.md

```diff
@@ -60,13 +60,13 @@ Next, we will do something with the image:
 
 ```python
 fig, ax = plt.subplots()
-plt.imshow(chair)
+ax.imshow(chair)
 ```
 
 Once we have the image in the program,
 we first call `plt.subplots()` so that we will have
 a fresh figure with a set of axis independent from our previous calls.
```

Comment:
```suggestion
a fresh figure with a set of axes independent from our previous calls.
```
This doesn't come from the current PR, but I guess the original author intended 'axes' plural, right?